### ISSUE TITLE
[Package Request] - Upgrade GNU nano

### ISSUE THREAD

**Issue by elsaco** [amazonlinux/amazon-linux-2023#711]
`nano-8.0` was released with modern key binding, many new features and bug fixes. AL2023 is still on version 5.8.

**Comment by stewartsmith** [comment]
If the build dependencies still work out okay, it's almost certainly a safe version bump. Thanks for letting us know there's interest!

**Comment by daniejstriata** [comment]
I built the newly released version from Fedora on COPR and had not problems building it for AL2023.

**Comment by walkerever** [comment]
did the same. built 1 on copr.

**Comment by stewartsmith** [comment]
We are looking at including a `nano` update in an upcoming release.

**Comment by danie-dejager** [comment]
Nano 8.3 was added?

**Comment by stewartsmith** [comment]
Yep! We are now at a nice new version of `nano` as of AL2023.7 (although I think it went out one release before the first .7 release...)
